bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -238,24 +252,22 @@ export class UIEvents
     @_trigger('rotate:end', e)
 
   _mouse_enter: (e) ->
-    @_bokify_jq(e)
-    @_trigger('move:enter', e)
+    @_bokify_point_event(e)
+    @trigger('move:enter', e)
```

Comment:
This looks like an unintentional change. As far as I know it should still be ``@_trigger()``.